app/src/main/java/com/sdpteam/connectout/validation/EditProfileValidator.java

```diff
@@ -34,30 +36,20 @@ public static boolean isValidBio(String bio) {
     }
 
     public static boolean editProfileValidation(EditText nameInput, EditText emailInput, EditText bioInput, RadioButton male, RadioButton female, RadioButton other) {
-        if(!isValidName(nameInput.getText().toString())) {
-            nameInput.setError(NAME_ERROR);
-            nameInput.requestFocus();
-            return false;
-        }
-        if(!isValidEmail(emailInput.getText().toString())) {
-            emailInput.setError(EMAIL_ERROR);
-            emailInput.requestFocus();
-            return false;
-        }
-        if(!isValidBio(bioInput.getText().toString())) {
-            bioInput.setError(BIO_ERROR);
-            bioInput.requestFocus();
+
```

Comment:
I noticed that the isValidPassword is only used in tests, I don't see where it could be needed, can you clarify its purpose ?

Reply:
okay well, I can delete it, it was for the case we have a "change password" functionality in the future, but I get it, ill just delete it

Reply:
I agree with your suggestion, this functionality seems practical !
It should added it to the backlog.